src/tools/rbd_wnbd/wnbd_handler.cc

```diff
@@ -367,6 +369,28 @@ void WnbdHandler::LogMessage(
           << WnbdLogLevelToStr(LogLevel) << " " << Message << dendl;
 }
 
+int WnbdHandler::resize(uint64_t new_size)
+{
+  int err = 0;
+
+  uint64_t new_block_count = new_size / RBD_WNBD_BLKSIZE;
```

Comment:
Why is `RBD_WNBD_BLKSIZE` hard-coded here?  Can `WnbdHandler::block_size` field be used instead?